db/migrate/20160928080048_rearrange_listing_uuid_bytes.rb

```diff
@@ -1,14 +1,7 @@
 class RearrangeListingUuidBytes < ActiveRecord::Migration
   def up
```

Comment:
Hm. Does this work? Should it have `down` also? Or is it inherited from ActiveRecord::Migration as no-op?

Other option could be `change`.